### PR TITLE
[NOMERGE] New package: tunesviewer

### PR DESCRIPTION
**Pull request by cr6git**
[#8451](https://github.com/voidlinux/void-packages/issues/8451).

### CODE REVIEW

File: srcpkgs/tunesviewer/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'tunesviewer'
+pkgname=tunesviewer
+version=1.5.4
+revision=1
+only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/tunesviewer/template]: why?

**Comment by Vaelatern** [comment]
This is not a source build for an AGPL project?

**Comment by cr6git** [comment]
@Vaelatern would you agree with `license="AGPL-2"` ?

**Comment by cr6git** [comment]
No response? I close.

**Comment by Vaelatern** [comment]
I'm awful about checking my git notifications. With that license, I can accept `GPL-2.0`

**Comment by cr6git** [comment]
Unfortunately, the latest release 2.1 does not build. 😕

**Pull request closed without merging**